src/lib/peer.ts

```diff
@@ -27,7 +27,9 @@ export class PeerAPI extends BaseAPI {
     }
 
     this.cache[address] = this.lambdasClient
-      .fetchProfiles([address.toLowerCase()])
+      .fetchProfiles([address.toLowerCase()], undefined, {
+        headers: { 'Cache-Control': 'max-age=0' }
```

Comment:
@juanmahidalgo you may also want to add `no-cache`

https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.32

Reply:
sure, will do it in the following PR!